openpgp.d.ts

```diff
@@ -22,6 +22,8 @@ export function generateSessionKey(options: { encryptionKeys: PublicKey[], date?
 export function decryptKey(options: { privateKey: PrivateKey; passphrase?: string | string[]; config?: PartialConfig }): Promise<PrivateKey>;
 export function encryptKey(options: { privateKey: PrivateKey; passphrase?: string | string[]; config?: PartialConfig }): Promise<PrivateKey>;
 export function reformatKey(options: { privateKey: PrivateKey; userIDs?: UserID|UserID[]; passphrase?: string; keyExpirationTime?: number; config?: PartialConfig }): Promise<KeyPair>;
+export function revokeKey(options: { key: PrivateKey, reasonForRevocation?: { flag?: enums.reasonForRevocation, string?: String }, config?: PartialConfig }): Promise<PrivateKey>;
+export function revokeKey(options: { key: PublicKey, revocationCertificate?: String, config?: PartialConfig }): Promise<PublicKey>;
```

Comment:
These will come in #1324, could you leave them out of this PR for now?

Reply:
Yup, that's because I merged from my master